### ISSUE TITLE
Error on using scan select accounts

### ISSUE THREAD

**Issue by jackrakan12** [rmcrackan/Libation#559]
**Describe the bug**
Libation does not show new books on auto scan and when using scan select accounts it does not being up new books and comes up an error message 
**To Reproduce**

1. Go to '...' import
2. Click on '....' scan library for some accounts
3. Scroll down to '....' an account to scan and hit scan

**Expected behavior**

**Screenshots**

**Platform**
windows 10
[e.g. Windows 10, Windows 11, Mac, Linux (State distribution)]

**Log Files**
Attach your Libation log file here.

[_Log202203.log](https://github.com/rmcrackan/Libation/files/11104277/_Log202203.log)

**Comment by Mbucari** [comment]
You're using version 6.7.3, which is no longer supported.

Please download the latest version [from here](https://github.com/rmcrackan/Libation/releases/tag/v9.4.7).

**Comment by rmcrackan** [comment]
fyi this link will always point to the latest full release: https://github.com/rmcrackan/Libation/releases/latest

You can get it by going to the latest full release's page. It's the link on the green "Latest" tag

**Comment by jackrakan12** [comment]
Hi thanks for the quick reply.  I have deleted the old version and installed the latest and having the same problem, should I go through the same process.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by rmcrackan** [comment]
Thanks. Please delete the old log, reproduce the error, then post the new log here.

**Comment by jackrakan12** [comment]
Sorry to bother you but the new log is 290,789kb and too big to upload

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by Mbucari** [comment]
@jackrakan12 So sorry. This is a bug that's going to be fixed within the next few hours.

**Comment by rmcrackan** [comment]
> This is a bug that's going to be fixed within the next few hours

Oh, I wasn't intending to release a new version for a bit since we just released one. Sounds like this bug needs fixing sooner rather than later though.

**Comment by Mbucari** [comment]
@rmcrackan Yeah, there are pretty important bug fixes pending. Some people can't scan their accounts right now because of the orphaned episode problem.

**Comment by rmcrackan** [comment]
Fixed in v10. Released today
